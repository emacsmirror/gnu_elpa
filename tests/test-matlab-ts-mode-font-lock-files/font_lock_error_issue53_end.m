% -*- matlab-ts -*-

% See https://github.com/acristoffers/tree-sitter-matlab/issues/53

v  = 1:10
g1 = v(2:end)
g2 = v(end:end)
g3 = v(end-2)
g4 = v(end+2-3)

a = 1:10

b = a(end)
c = a(end, :)
d = a(end, end)

x = [1, 2
     3, 4];
y = [5, 6
     7, 8];
z = [9, 10
     11, 12];

xyz = cat(3, x, y, z)

x1 = xyz(end, :, :)
x2 = xyz(end, end, :)
x3 = xyz(:, :, end)
x4 = xyz(end, end, end)
