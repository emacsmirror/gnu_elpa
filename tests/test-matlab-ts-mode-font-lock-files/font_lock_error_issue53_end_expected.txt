c ccc ccccccccc ccc

c ccc cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc

v  o nDnn
vv o dbnDNNNb
vv o dbNNNDNNNb
vv o dbNNNonb
vv o dbNNNononb

v o nDnn

v o dbNNNb
v o dbNNND Db
v o dbNNND NNNb

v o bnD n
     nD nbD
v o bnD n
     nD nbD
v o bnD nn
     nnD nnbD

vvv o BBBbnD dD dD db

vv o dddbNNND DD Db
vv o dddbNNND NNND Db
vv o dddbDD DD NNNb
vv o dddbNNND NNND NNNb
